share/python/g3mp2.py

```diff
@@ -0,0 +1,685 @@
+                   [0.0, 0.0],         # 25 Mn    Manganese
+                   [0.0, 0.0],         # 26 Fe    Iron
+                   [0.0, 0.0],         # 27 Co    Cobalt
+                   [0.0, 0.0],         # 28 Ni    Nickel
+                   [0.0, 0.0],         # 29 Cu    Copper
+                   [0.0, 0.0],         # 30 Zn    Zinc
+                   [65.00, 65.00],     # 31 Ga    Gallium
+                   [88.91, 88.91],     # 32 Ge    Germanium
+                   [73.90, 72.42],     # 33 As    Arsenic
+                   [55.76, 54.27],     # 34 Se    Selenium
+                   [26.74, 28.18],     # 35 Br    Bromine
+                   [0.0, 0.0]]         # 36 Kr    Krypton
+
+        if Z < 1 or Z >= len(atomDHF):
+            raise ValidationError("dHf_one_atom: Invalid atomic number %d" % Z)
+        else:
+            return (atomDHF[Z][0], atomDHF[Z][1])
+
+    def heat_of_formation(_E0, _H298):
+        '''
+        calculate heats of formation at
+        0K and 298K by atomization method
+
+        input:    ab initio E0, H298 in Hartrees
+        returns: dHf@0K, dHf@298K -> 2 doubles
+        '''
+
+        debug("heat_of_formation(%.6f, %.6f) entry" % (_E0, _H298))
+
+        E0_sum_atoms = 0.0
+        H298_sum_atoms = 0.0
+        dhf0_sum_atoms = 0.0
+        dhf298_sum_atoms = 0.0
+
+        _natoms = mol.natom()
+
+        for A in range(0, _natoms):
+            Z = int(mol.Z(A))
+
+            e0_atom, h298_atom = E0_one_atom(Z)     # in Hartrees
+            E0_sum_atoms += e0_atom
+            H298_sum_atoms += h298_atom
+
+            d0, d298 = dHf_one_atom(Z)              # in kcal/mol
+            dhf0_sum_atoms += d0
+            dhf298_sum_atoms += d298
+
+        dhf0 = (_E0 - E0_sum_atoms) * psi_hartree2kcalmol + dhf0_sum_atoms
+        dhf298 = (_H298 - H298_sum_atoms) * psi_hartree2kcalmol + dhf298_sum_atoms
+
+        # Eatomization = -(_H298 - H298_sum_atoms) * psi_hartree2kcalmol
+
+        return dhf0, dhf298
+
+    # ______________________________________________________
+    # _____________________      ___________________________
+    # _____________________ MAIN ___________________________
+    # ______________________________________________________
+
+    nAtoms = 0
+    charge = 0
+    multiplicity = 1
+
+    kwargs = p4util.kwargs_lower(kwargs)
+    debug('%s: %d args: %s' %
+          (name, len(kwargs), ','.join([arg for arg in kwargs])))
+
+    flag_list = ["verbose", "qcisdt", "debug", "help"]
+    arg_list = list(kwargs)
+
+    if "help" in kwargs:
+        g3mp2_usage()
+        return 0.0      # quit
+
+    for arg in arg_list:
+        if arg in flag_list:
+
+            if arg == 'verbose':
+                g3mp2_verbose_flag = kwargs.pop('verbose', True)
+
+            if arg == 'qcisdt':
+                use_QCISDT = kwargs.pop('qc == dt', True)
+
+            if arg == 'debug':
+                kwargs.pop('debug')
+                verbose()
+                set_debug(1)
+
```

Comment:
The above loop is not needed. 

`g3mp2_verbose_flag = kwargs.pop('verbose', True)` will search kwargs for `verbose` and return that if it exists else it will return `true`.